version/version.go

```diff
@@ -4,5 +4,5 @@
 
 package version
 
-const defaultBeatVersion = "8.12.0"
+const defaultBeatVersion = "8.13.0"
```

Comment:
We need to change the tests to keep creating 8.12 stacks until the first 8.13 snapshot is ready. Don't merge this until that is done.